reactive/docker.py

```diff
@@ -130,6 +130,8 @@ def install():
         set_state('nvidia-docker.installed')
     elif runtime == "apt":
         install_from_archive_apt()
+    elif runtime == "custom":
+        install_from_custom_apt()
```

Comment:
What if `install_from_custom_apt()` returns `False` without having actually installed docker? It looks like this will continue on under the assumption that docker is now installed, which is not necessarily the case.